Public/Src/App/Bxl/BuildXLApp.cs

```diff
@@ -950,8 +950,10 @@ private AppResult RunWithLoggingScope(Func<PerformanceMeasurement, AppResult> ru
                     {
                         Stopwatch sw = Stopwatch.StartNew();
                         Exception telemetryShutdownException;
-
-                        var shutdownResult = AriaV2StaticState.TryShutDown(out telemetryShutdownException);
+                        
+                        // Allow a longer Aria telemetry flush time in CloudBuild since we're more willing to wait at the tail of builds there
+                        TimeSpan telemetryFlushTimeout = m_configuration.InCloudBuild() ? TimeSpan.FromMinutes(1) : AriaV2StaticState.DefaultShutdownTimeout;

```

Comment:
can you move this to a property so that it can easily be passed to `AriaV2StaticState.CreateAriaLogger`?